src/ApiMethodDocumentation.js

```diff
@@ -1014,7 +1014,7 @@ export class ApiMethodDocumentation extends AmfHelperMixin(LitElement) {
         ?opened="${callbacksOpened}"
       >
         <div class="heading3 table-title" role="heading" aria-level="2">Callbacks</div>
-        <div class="title-area-actions" aria-label="${buttonState}">
+        <div class="title-area-actions" aria-label="${buttonState}" data-toggle="callbacks">
```

Comment:
Should this `data-toggle` be applied to the `anypoint-button` instead of the `div`?

Reply:
You're right. Done!